src/services/ceramic-service.ts

```diff
@@ -27,8 +27,25 @@ export default class CeramicServiceImpl implements CeramicService {
   }
 
   async loadDocument<T extends Doctype>(docId: DocID): Promise<T> {
+    let timeout: NodeJS.Timeout;
+
     const docPromise = this._client.loadDocument(docId, {sync: false})
-    const timeoutPromise = new Promise((_, reject) => setTimeout(() => reject(`Timed out loading docid: ${docId.toString()}`), 60 * 1000))
+      .then((data) => {
+        return data;
+      })
+      .catch((err) => {
```

Comment:
Haha, now that you're using `finally` I don't think you need the `.then()` or `.catch()` cases at all

Reply:
@stbrody Still do to return data and catch errors. Finally doesn't receive any args.

Reply:
test.js:
```
const foo = async function() {
    return "foo";
}

const runTest = async function() {
    const prom = foo()
    prom.finally(() => {
        console.log('finally called')
    });

    const res = await prom
    console.log("result: " + res)
}

runTest()
```

```
> node test.js
finally called
result: foo
```

Reply:
Finally just transforms the Promise into one that runs the callback before returning, but it doesn't change the way the resolved value or error passes through

Reply:
Ahh I see. Updated